R/theme-current.R

```diff
@@ -64,6 +68,14 @@ NULL
 #'
 #' # theme_update() and theme_replace() are similar except they
 #' # apply directly to the current/active theme.
+#'
+#' # `theme_set()` completely overrides the current theme.
+#' # `theme_update()` modifies a particular element of the current theme.
+#' theme_set(theme_classic())
```

Comment:
A similar example is already given further up in the examples. Please move this entire block up and merge with the previous example of `theme_set()`. Also, it's important to undo any changes to the default theme at the end of the example.